arduino-ide-extension/src/browser/boards/boards-toolbar-item.tsx

```diff
@@ -220,12 +323,21 @@ export class BoardsToolBarItem extends React.Component<
       'fa',
       protocolIcon
     );
-
+    const originalOnSelect = boardList.onSelect;
+    boardList['onSelect'] = (params) => {
+      this.hide();
+      return originalOnSelect.bind(boardList)(params);
```

Comment:
Seeing `bind` in the `render` method feels unfamiliar to me, considering the [docs](https://legacy.reactjs.org/docs/faq-functions.html#how-do-i-bind-a-function-to-a-component-instance) "Using Function.prototype.bind in render creates a new function each time the component renders, which may have performance implications (see below).", could this be done more efficiently elsewhere?